Source/WTF/Scripts/generate-platform-args

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+import subprocess
+import os
+import shlex
+import shutil
+import re
+
+# Preprocesses wtf/Platform.h with clang, extracts macro definitions, and saves
+# the platform flags to a Swift-style response file containing -D arguments.
```

Comment:
Maybe the script name should include "swift" somehow.

Reply:
There are additional places where we generate listings of platform flags like this (e.g. WebCore below) plus some locations where we compute the search path flags. I'd like to make this script handle those cases as well, in a follow up, so I intended for the script's name to be not-swift-specific.

I'll leave a fixme for those use cases.